app/assets/javascripts/app/views/single-post-viewer/single_post_content_view.js

```diff
@@ -23,6 +23,13 @@ app.views.SinglePostContent = app.views.Base.extend({
     this.pollView = new app.views.Poll({ model: this.model });
   },
 
+  postRenderSubviews: function() {
+    var photoAttachments = this.$(".photo_attachments");
+    if(photoAttachments.length > 0) {
+      new app.views.Gallery({ el: photoAttachments });
+    }
+  },
```

Comment:
I'd say instead of adding this here you could add this to the `postRenderTemplate` function of `app.views.ExpandedStatusMessage` (in content_view.js). Then you could also remove `postRenderSubviews` from views.js.